apps/comps/components/user-info/ProfilePicture.tsx

```diff
@@ -89,12 +89,12 @@ export const ProfilePicture: React.FC<ProfilePictureProps> = ({
       tabIndex={0}
       aria-label="Change profile picture"
     >
-      <div className="flex flex-col items-center px-2 py-4">
-        <span className="flex gap-2 text-xs font-medium">
-          <SquarePen className="text-secondary-foreground mr-2 inline-block" />
-          Profile Picture URL
-        </span>
-        <span className="text-secondary-foreground mt-1 text-xs">
+      <div className="flex flex-col items-center gap-1 px-2 py-4">
```

Comment:
general fixups related to "edit"; i didnt put a tooltip here because the position of the tip doesnt work nicely on the hovered div

<img width="315" height="279" alt="Image" src="https://github.com/user-attachments/assets/81d66a61-3204-4b3c-95e4-fc1b731562e1" />